test/cql-pytest/test_compactionstats.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2024-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+import pytest
+
+import nodetool
+from util import new_test_table
+
+
+def get_compaction_stats(cql, table):
+    ks, cf = table.split('.')
+    stats = nodetool.compactionstats(cql)
+    tasks = 0
+    for s in stats:
+        if s['ks'] == ks and s['cf'] == cf:
+            tasks += int(s['task'])
+    return tasks, stats
+
+@pytest.mark.parametrize("compaction_strategy", ["LeveledCompactionStrategy", "SizeTieredCompactionStrategy", "TimeWindowCompactionStrategy"])
+def test_compactionstats_after_major_compaction(scylla_only, cql, test_keyspace, compaction_strategy):
+    """
+    Test that compactionstats show no pending compaction after major compaction
+    """
+    num_sstables = 16
+    extra_strategy_options = ""
+    if compaction_strategy == "LeveledCompactionStrategy":
+        extra_strategy_options = ", 'sstable_size_in_mb':1"
+        num_sstables *= 4   # Need enough data to trigger level 0 compaction
+    value = 'x' * 128*1024
```

Comment:
Another question regarding the potential slowness of this test: why do you need such large writes?

Reply:
See https://github.com/scylladb/scylladb/pull/20892#discussion_r1781684728
I didn't benchmark the test but the i/o part of it doesn't seem ti be significant.